ci/deploy.sh

```diff
@@ -64,7 +64,8 @@ fi
 # Fetch and create gh-pages and output branches
 # Travis does a shallow and single branch git clone
 git remote set-branches --add origin gh-pages output
-git fetch origin gh-pages:gh-pages output:output
+git fetch origin gh-pages:gh-pages output:output || \
+  echo >&2 "[INFO] could not fetch gh-pages or output from origin."
```

Comment:
I finally am starting to understand bash conditionals haha. This article helped https://www.linuxjournal.com/content/understanding-bash-elements-programming